app/src/main/java/com/example/concordia_campus_guide/Models/Routes/TransportType.java

```diff
@@ -0,0 +1,4 @@
+package com.example.concordia_campus_guide.Models.Routes;
+
+public abstract class TransportType {
```

Comment:
Is there an attribute that is common to all transport types that we could put in this abstract class with an abstract method? Like getDuration()?

Reply:
nope or else I would've had it

Reply:
ok sounds good